app/Connect/Support/BaseApiAction.php

```diff
@@ -26,16 +26,16 @@ public function handleRequest(Request $request): JsonResponse
 
     protected function buildResponse(array $result): JsonResponse
     {
-        if (array_key_exists('Status', $result)) {
-            $status = $result['Status'];
-            if ($status === 401) {
-                return response()->json($result, $status)->header('WWW-Authenticate', 'Bearer');
-            }
+        $status = $result['Status'] ?? 200;
+        $response = response()->json($result, $status);
 
-            return response()->json($result, $status);
+        $response->header('Content-Length', (string) strlen($response->getContent()));
```

Comment:
https://retroachievements.org/forums/topic/31122?comment=294115#294115

Unfortunately, it appears adding the "Content-Length" header didn't remediate the issue.